.github/workflows/emberjs-addons-tag-release-publish.yml

```diff
@@ -0,0 +1,83 @@
+# GitHub Actions workflows:
+# https://docs.github.com/en/actions
+
+name: Create new `git tag`, create new GitHub release and publish to NPM
+
+on:
+  workflow_call:
+    inputs:
+      nodejs_version:
+        description: Version of Node.js to use.
+        default: '12'
+        required: false
+        type: string
+
+    secrets:
+      npm_automation_token:
+        description: |
+          NPM automation token authorised to publish versions.
+          Check the settings of your package on npm.
+        required: true
+
+jobs:
+  create_git_tag:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    name: Create new `git tag`
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    outputs:
+      new_tag: ${{ steps.detect_then_tag.outputs.tag }}
+      new_version: ${{ steps.detect_then_tag.outputs.current-version }}
+      old_version: ${{ steps.detect_then_tag.outputs.previous-version }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Required to ensure git history is properly checked:
+          # https://github.com/salsify/action-detect-and-tag-new-version/blob/v2.0.1/README.md?plain=1#L11
+          fetch-depth: 2
+
+      - name: Detect and tag new version
+        id: detect_then_tag
+        uses: salsify/action-detect-and-tag-new-version@v2
+
+  create_github_release:
+    if: ${{ needs.create_git_tag.outputs.new_tag }}
+    name: Create new GitHub release
+    needs: create_git_tag
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: ncipollo/release-action@v1
+        with:
+          body: "\
+            Changelog: \
+            ${{ github.server_url }}/${{ github.repository }}\
+            /compare/\
+            v${{ needs.create_git_tag.outputs.old_version }}...v${{ needs.create_git_tag.outputs.new_version }}\
+          "
```

Comment:
Why this syntax? => In order to split a long string over multiple lines, without getting a whitespace between each line.

See:
https://stackoverflow.com/questions/6268391/is-there-a-way-to-represent-a-long-string-that-doesnt-have-any-whitespace-on-mul

In comparison to:

```yaml
body: >
  Changelog: 
  my_repo
  /compare/
  v1...v2
```

Which would result in the following string: (notice the whitespaces between each section)

```
Changelog: my_repo /compare/ v1...v2
```